DigitalLearningSolutions.Data/Services/CourseService.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Models.Courses;
+
+    public interface ICourseService
+    {
+        IEnumerable<CourseStatistics> GetTopCourseStatisticsAtCentreForCategoryId(int centreId, int categoryId);
+    }
+
+    public class CourseService : ICourseService
+    {
+        private readonly ICourseDataService courseDataService;
+
+        public CourseService(ICourseDataService courseDataService)
+        {
+            this.courseDataService = courseDataService;
+        }
+
+        public IEnumerable<CourseStatistics> GetTopCourseStatisticsAtCentreForCategoryId(int centreId, int categoryId)

```

Comment:
I think the name "GetTopCourseStatistics" would be sufficient as the centreId & cetegoryId are already present in params (just to make the name a bit easier to read).